src/pages/WizzardPage2/WizzardPage2.js

```diff
@@ -0,0 +1,504 @@
+import PropTypes from "prop-types";
+
+import classes from "./WizzardPage2.module.scss";
+import Input from "../../components/Wizzard/Input/Input";
+import Select from "../../components/Wizzard/Select/Select";
+import useInput from "../../hooks/useInput";
+import WizzardFormLayout from "../../layout/WizzardFormLayout/WizzardFormLayout";
+import WizzardButtons from "../../components/Wizzard/WizzardButtons/WizzardButtons";
+import { Contex } from "../../store/Store";
+import useAxios from "../../hooks/useAxios";
+import {
+  WIZARD_PAGE_2_EMAIL_LABEL,
+  WIZARD_PAGE_2_EMAIL_ERROR_MESSAGE,
+  WIZARD_PAGE_2_ADDRESS_1_ERROR_MESSAGE,
+  WIZARD_PAGE_2_ADDRESS_1_LABEL,
+  WIZARD_PAGE_2_ADDRESS_2_LABEL,
+  WIZARD_PAGE_2_COUNTRY_LABEL,
+  WIZARD_PAGE_2_COUNTRY_ERROR_MESSAGE,
+  WIZARD_PAGE_2_CITY_LABEL,
+  WIZARD_PAGE_2_CITY_ERROR_MESSAGE,
+  WIZARD_PAGE_2_POSTAL_CODE_LABEL,
+  WIZARD_PAGE_2_POSTAL_CODE_ERROR_MESSAGE,
+  WIZARD_PAGE_2_HOW_LONG_LIVED_LABEL,
+  WIZARD_PAGE_2_HOW_LONG_LIVED_ERROR_MESSAGE,
+} from "../../static";
+
+/**
+ * Second page that users lands on after successfully submitting data in first page
+ *
+ * @param {object} props A function handling wizard rendering path
+ * @returns {JSX} Wizard page two form
+ */
+
+const WizzardPage2 = (props) => {
+  const { onForm2Submit, setStep } = props;
+  const ctx = useContext(Contex);
+  const wizardData = ctx.state;
+
+  const [state, setState] = useState({
+    email: wizardData.email,
+    address1: wizardData.address1,
+    address2: wizardData.address2,
+    country: wizardData.country,
+    city: wizardData.city,
+    postalCode: wizardData.postalCode,
+    howManyYearsLived: wizardData.howManyYearsLived,
+  });
+
+  const [countries, setCountries] = useState(wizardData.countries);
+  const [selectedCountries, setSelectedCountries] = useState(
+    wizardData.selectedCountries
+  );
+  const [selectedStateTLA, setSelectedStateTLA] = useState(
+    wizardData.selectedStateTLA
+  );
+
+  const [selectedCities, setSelectedCities] = useState(
+    wizardData.selectedCities
+  );
+
+  const [postalCodes, setPostalCodes] = useState(wizardData.postalCodes);
+  const [selectedPostalCodes, setSelectedPostalCodes] = useState(
+    wizardData.selectedPostalCodes
+  );
+  const [selectedPostalCodeNumber, setSelectedPostalCodeNumber] = useState(
+    wizardData.selectedPostalCodeNumber
+  );
+
+  const { fetchData: getStates } = useAxios(
+    "http://det.api.rs.ey.com/api/states"
+  );
+
+  useEffect(() => {
+    getStates(setCountries);
+  }, [getStates]);
+
+  const { fetchData: getCities } = useAxios(
+    `http://det.api.rs.ey.com/api/states/${selectedStateTLA}/cities`
+  );
+
+  const { fetchData: getPostalCodes } = useAxios(
+    `http://det.api.rs.ey.com/api/states/${selectedStateTLA}/cities/${state.city}/postalcodes`
+  );
+
+  useEffect(() => {
+    if (state.city) {
+      getPostalCodes(setPostalCodes);
+    }
+  }, [state.city, getPostalCodes]);
+
+  useEffect(() => {
```

Comment:
If there is some complex code inside useEffect hook, move the code to separate function, and invoke it inside useEffect body.
It is ok if you leave it here like this, but bear this in mind for the future.